### DISCUSSION TITLE
Using GCN on partly labeled (including missing features) graph data

### DISCUSSION THREAD

**Discussion by kingasiuta** [pyg-team/pytorch_geometric#4255] [Q&A]
I am trying to use the framework to make node classification on a large graph that is only partly labeled also in terms of features (i.e., not just the target variable). There are 200k nodes in the network, out of which 20k have features and labels assigned to them. The other ones are simply vertices in the graph that we know nothing about except for the edges that go through them. Can such a network be used for the node classification task in a way that does not result in losing information about the connections within the network?

**Comment by rusty1s** [comment]
Yes, it can. There exists some recent exciting work to let GNNs operate on graphs with missing features. We try to integrate one of those methods into the library here: https://github.com/pyg-team/pytorch_geometric/issues/4176
